.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: All
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUSTFLAGS: '-D warnings'
+
+jobs:
+  test-stable:
+    name: Test 32/64 bit stable
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          # 32-bit target
```

Comment:
Do you think it's worth testing on 32 bit? Seems unlikely that it gets any substantial usage these days.

Reply:
Yeah yup nah I've removed it.